app/javascript/article-form/components/ClipboardButton.jsx

```diff
@@ -27,7 +27,7 @@ export const ClipboardButton = ({
       onClick={onCopy}
       className="spec__image-markdown-copy whitespace-nowrap fw-normal"
       icon={CopyIcon}
-      title="Copy markdown for image"
```

Comment:
With the test tool updated, this spec is consider a failure because there's no way to see the whole title 
![image](https://user-images.githubusercontent.com/15793250/204329002-646165b3-8905-4505-8c96-fbf4c52cbe5a.png)
I'm tested on both desktop and mobile view; the content is too long. So I've simplified it.